addons/mail/models/mail_scheduled_message.py

```diff
@@ -150,6 +152,29 @@ def write(self, vals):
             self.env.ref('mail.ir_cron_post_scheduled_message')._trigger(fields.Datetime.to_datetime(new_scheduled_date))
         return res
 
+    # ------------------------------------------------------
+    # Compute Methods
+    # ------------------------------------------------------
+
+    @api.depends('model', 'partner_ids', 'res_id')
+    def _compute_notified_bcc(self):
+        """ When being in monorecord comment mode, compute 'bcc' which are
+        followers that are going to be 'silently' notified by the message. """
+        for composer in self:
+            record = self.env[composer.model].browse(composer.res_id)
```

Comment:
You should probably be batch-enabled: group by model, prefetch ids, ... I fear people end up with issue if somehow this field has to be computed sequentially on a lot of records.